internal/librarian/generate.go

```diff
@@ -506,6 +506,9 @@ func (r *generateRunner) runConfigureCommand(ctx context.Context) (string, error
 	if err != nil {
 		return "", err
 	}
+	if libraryState == nil {
+		return "", fmt.Errorf("no response file for configure container command")
```

Comment:
```suggestion
		return "", fmt.Errorf("no response file for generate container command")
```

Reply:
This is within `runConfigureCommand` - it would be the generate *CLI* command, but it's the configure *container* command.